docs/operator-guide/upgrade/upgrade-edp-3.10.md

```diff
@@ -0,0 +1,347 @@
+      # -- Defines Keycloak client name that is used for the Identity Provider (IdP) client
+      ssoClientName: "edp"
+      # -- Keycloak URL.
+      keycloakUrl: https://keycloak.example.com/auth
+      # -- Administrators of your tenant.
+      admins:
+        - "stub_user_one@example.com"
+      # -- Developers of your tenant
+      developers:
+        - "stub_user_one@example.com"
+        - "stub_user_two@example.com"
+
+      # Oauth2-proxy image
+      image:
+        # -- OAuth2-proxy image repository
+        repository: quay.io/oauth2-proxy/oauth2-proxy
+        # -- OAuth2-proxy image tag
+        tag: v7.4.0
+
+      # Create a cookie-secret with the following command:
+      # 'openssl rand -base64 32 | head -c 32 | base64'
+      # Use an existing secret for OAuth2 cookie-secret
+      existingSecret:
+        # -- Secret name which stores cookie-secret
+        secretName: oauth2-proxy-cookie-secret
+        # -- Secret key which stores cookie-secret
+        secretKey: cookie-secret
+
+      # -- Additional container environment variables
+      extraEnv: []
+      # -- Extra arguments to provide to the OAuth2-proxy
+      extraArgs: {}
+
+      # -- Additional volumes to be added to the OAuth2-proxy pod
+      extraVolumes: []
+      #  - name: custom-ca
+      #    secret:
+      #      defaultMode: 420
+      #      secretName: custom-ca
+
+      # -- Additional volumeMounts to be added to the OAuth2-proxy container
+      extraVolumeMounts: []
+      #  - name: custom-ca
+      #    mountPath: /etc/ssl/certs/CA.crt
+      #    readOnly: true
+      #    subPath: CA.crt
+
+      ingress:
+        # -- Enable ingress controller resource
+        enabled: true
+        # -- Additional ingress annotations
+        annotations: {}
+        # -- Ingress path type. One of `Exact`, `Prefix` or `ImplementationSpecific`
+        pathType: Prefix
+
+        # For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
+        # Ref: https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#specifying-the-class-of-an-ingress
+        # -- Defines which ingress controller will implement the resource, e.g. nginx
+        ingressClassName: ""
+        # -- Ingress TLS configuration
+        tls: []
+        #  - secretName: chart-example-tls
+        #    hosts:
+        #      - chart-example.local
+      # -- Node labels for pod assignment
+      nodeSelector: {}
+      # -- Toleration labels for pod assignment
+      tolerations: []
+      # -- Affinity settings for pod assignment
+      affinity: {}
+      ```
+
+      </details>
+
+    If you previously used SSO configuration section, follow these migration steps:
+
+    1. Manually remove all Keycloak resources previously created by the **edp-install** chart. The list of resources to remove can be found [here](https://github.com/epam/edp-install/tree/v3.9.0/deploy-templates/templates/keycloak).
```

Comment:
why? helm do not remove this objects from cluster?

Reply:
yes, helm will not delete resources. All Keycloak resources will be deleted at the same time, which may lead to errors during deletion.